### PR TITLE
Tests fixtures redefined and single commit

### LINKED ISSUE #3: Implement sessions() methods
Need to implement methods from [sessions](https://www.w3.org/TR/webdriver/#sessions) section of the standard.
* [x] Create session (`driver = WebDriver()`)
* [x] `driver.sessions().delete()`
* [x] `status = driver.sessions().status` using `@property`

### PR DESCRIPTION
**Pull request by pychik**
fixes #3 #5

### CODE REVIEW

**Comment by pychik** [comment]
This PR is a propose to use fixture way to divide client settings like firefox_client, chrome_client and other in tests, really convenient.
Also tested rebase method

File: tests/functional/config.py [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-    CAPABILITIES = {'browser_name': 'firefox', 'selenoidoptions': {"enableVnc": True, "enableVideo": False}}
-    TEST_CAPS: dict = {
-        "browserName": "firefox",
-        "acceptInsecureCerts": True,
-        "moz:debuggerAddress": True,
-        }
+    class Firefox:
+        CAPABILITIES = {'browserName': 'firefox', 'selenoid:options': {"enableVnc": True, "enableVideo": False}}
+        # CAPABILITIES = Capabilities(browser_name='firefox', selenoidoptions={"enableVnc": True, "enableVideo": False})
+        CONTAINER: dict = {
```
Comment (vania-pooh) [inline tests/functional/config.py]: Delete this field.

File: tests/functional/conftest.py [hunk 2]
```diff
@@ -19,20 +19,18 @@ def event_loop():
 
 
 @pytest.fixture(scope='session')
-def selenoid_container():
-    with BrowserWebDriverContainer(settings.test_caps) as firefox:
+def firefox_container():
+    with BrowserWebDriverContainer(settings.Firefox.CONTAINER) as firefox:
```
Comment (vania-pooh) [inline tests/functional/conftest.py]: Migrate to DockerContainer and use the following images:
```
CHROME("browsers/chrome:104.0”),
    EDGE("browsers/edge:104.0”),
    FIREFOX("browsers/firefox:103.0”),
    OPERA("browsers/opera:89.0”),
    SAFARI("browsers/safari:15.0");
```

File: tests/functional/src/test_sessions.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class TestSimple:
 
-    def test_session(self, selenoid_client):
+    def test_session(self, firefox_client):
         # get session_id
-        assert len(selenoid_client.sessions.id) > 10
+        assert len(firefox_client.sessions.id) > 10
```
Comment (vania-pooh) [inline tests/functional/src/test_sessions.py]: Magic number. Probably need to use `> 0`

**Review by vania-pooh — CHANGES_REQUESTED** [review]

File: tests/functional/src/test_sessions.py (hunk 3 above)
Comment (pychik) [reply tests/functional/src/test_sessions.py]: fixed in 9b20364a666f03413286df31a3d4c2cfb919f0c5

File: tests/functional/conftest.py (hunk 2 above)
Comment (pychik) [reply tests/functional/conftest.py]: fixed in 9b20364a666f03413286df31a3d4c2cfb919f0c5

File: tests/functional/config.py (hunk 1 above)
Comment (pychik) [reply tests/functional/config.py]: fixed in 9b20364a666f03413286df31a3d4c2cfb919f0c5

File: pages/script.js [hunk 4]
```diff
@@ -0,0 +1,444 @@
+/**
```
Comment (vania-pooh) [inline pages/script.js]: What is the use case of this file?

File: client/mixin.py [hunk 5]
```diff
@@ -53,7 +57,7 @@ def screenshot(self) -> Screenshot:
         return self._wd.screenshot
 
     @property
-    def context(self) -> Context:
+    def windows(self) -> Windows:
         return self._wd.context
```
Comment (vania-pooh) [inline client/mixin.py]: Renamed `context` -> `windows`.

File: ci/setter.sh [hunk 6]
```diff
@@ -21,4 +21,4 @@ pushd python/ && python setup.py install && cd ..
 flake8 client/ --count --select=E9,F63,F7,F82,E302,W391,W292 --show-source --statistics
```
Comment (vania-pooh) [inline ci/setter.sh:21]: Renamed this file to `ci/test.sh`.

**Review by vania-pooh — CHANGES_REQUESTED** [review]
Need to fix small issues. The main one is related to `script.js` file.

File: pages/script.js (hunk 4 above)
Comment (pychik) [reply pages/script.js]: only UI techno-lightnings on the background of page, just nice looking and good reference to naming. If it's not appropriate page decoration, and have no use-  i'm waiting for the words - **cleanse it!**

File: client/mixin.py (hunk 5 above)
Comment (pychik) [reply client/mixin.py]: Lack of ContextsApi tests and attentiveness.. Still need some assist in ContextsApi
naming fixed in 366fff82a1e6a5d423e57a770b430cf2306c4240

File: ci/setter.sh (hunk 6 above)
Comment (pychik) [reply ci/setter.sh:21]: fixed in 366fff82a1e6a5d423e57a770b430cf2306c4240

File: pages/script.js (hunk 4 above)
Comment (vania-pooh) [reply pages/script.js]: At least we need to make sure that this script file has appropriate license (Apache 2.0, MIT or BSD).

Comment (vania-pooh) [reply pages/script.js]: @pychik still need to fix this. I would remove this script if not strictly required.

Comment (pychik) [reply pages/script.js]: script and links removed in c0ca229daedede4d8347449befe7c9c4a131d574

**Review by vania-pooh — APPROVED** [review]

**Pull request merged**